btcwallet/btcwallet_test.go

```diff
@@ -0,0 +1,34 @@
+package btcwallet
+
+import (
+	"sync"
+	"testing"
+
+	_ "golang.org/x/tools/go/expect"
+)
+
+func TestBtcWallet(t *testing.T) {
+	wallet := NewBtcWallet(20)
```

Comment:
Even though NewBtcWallet() accepts float as an argument, let's pass a value of type `Bitcoin`

Reply:
Exactly, thank you